External versions: delete after 3 months of being merged/closed (#7678)
readthedocs/readthedocs.org: docs/guides/autobuild-docs-for-pull-requests.rst

This is a feature we plan to add,
but don't want to overwhelm our search indexes used in production.

> The documentation will be keep only for 90 days after the pull/merge request has been closed or merged.

Reviewer: ```suggestion
The built documentation is kept for 90 days after the pull/merge request has been closed or merged.
```
